src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -412,10 +412,13 @@ struct BlueprintParticleMesh
     the mesh is read in.
   */
   template <int NDIMS>
-  void setPoints(const axom::Array<primal::Point<double, NDIMS>>& pts)
+  void setPoints(axom::IndexType domainId,
+                 const axom::Array<primal::Point<double, NDIMS>>& pts)
   {
-    axom::IndexType domainIdx = createBlueprintStubs();
-    SLIC_ASSERT(m_domainGroups[domainIdx] != nullptr);
+    axom::IndexType localIdx = createBlueprintStubs();
+    SLIC_ASSERT(m_domainGroups[localIdx] != nullptr);
+    m_domainGroups[localIdx]->createViewScalar<std::int64_t>("state/domain_id",
+                                                             domainId);
```

Comment:
Minor: This function is also (potentially) mixing 32-bit and 64-bit ints w/ the conversion from `axom::IndexType` to `std::int64_t`

@cyrush -- when the blueprint refers to an `integer`, is there an assumption about its bitwidth?
See: https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html#state